### ISSUE TITLE
ImportError: cannot import name 'AsyncContextManager'

### ISSUE THREAD

**Issue by dghoshal-lbl** [aiidateam/kiwipy#75]
Hi,

I am receiving the following error once I run the provided tests after a successful installation. Let me know if I am missing anything.

```
../venv/lib/python3.6/site-packages/_pytest/config/__init__.py:495: in _importconftest
    return self._conftestpath2mod[key]
E   KeyError: PosixPath('/Users/dghoshal/kiwipy/test/rmq/conftest.py')

During handling of the above exception, another exception occurred:
../venv/lib/python3.6/site-packages/py/_path/local.py:704: in pyimport
    __import__(modname)
../venv/lib/python3.6/site-packages/_pytest/assertion/rewrite.py:152: in exec_module
    exec(co, module.__dict__)
test/rmq/conftest.py:6: in <module>
    from . import utils
test/rmq/utils.py:8: in <module>
    from kiwipy import rmq
kiwipy/rmq/__init__.py:2: in <module>
    from .communicator import *
kiwipy/rmq/communicator.py:10: in <module>
    import aio_pika
../venv/lib/python3.6/site-packages/aio_pika/__init__.py:1: in <module>
    from . import patterns, pool
../venv/lib/python3.6/site-packages/aio_pika/patterns/__init__.py:1: in <module>
    from .master import JsonMaster, Master, NackMessage, RejectMessage, Worker
../venv/lib/python3.6/site-packages/aio_pika/patterns/master.py:7: in <module>
    from aio_pika.channel import Channel
../venv/lib/python3.6/site-packages/aio_pika/channel.py:10: in <module>
    from .exchange import Exchange, ExchangeType
../venv/lib/python3.6/site-packages/aio_pika/exchange.py:8: in <module>
    from .message import Message
../venv/lib/python3.6/site-packages/aio_pika/message.py:10: in <module>
    from typing import (
E   ImportError: cannot import name 'AsyncContextManager'
```

**Comment by muhrin** [comment]
Thanks for the report @dghoshal-lbl .  I'm just looking into how this could have come about.

Would you mind posting the results of `pip freeze` so I can see the versions of libraries that you have installed?

**Comment by muhrin** [comment]
Could you also share your python version (`python --version`)?

The missing template (`AsyncContextManager`) has been available since python 3.5.4 and 3.6.2.  Is it possible that you have python < 3.6.2?

**Comment by dghoshal-lbl** [comment]
@muhrin You are correct. My python version was 3.6.1. After upgrading it, I was able to successfully run my test. Thanks!
